core/src/main/java/hudson/FilePath.java

```diff
@@ -1762,6 +1773,8 @@ public Void invoke(File f, VirtualChannel channel) throws IOException, Interrupt
                 try (InputStream fis = Files.newInputStream(reading(f).toPath());
                      OutputStream out = p.getOut()) {
                     org.apache.commons.io.IOUtils.copy(fis, out);
+                } catch (InvalidPathException e) {
```

Comment:
OK. Thought this was purely for logging, but seems it is actually a consumable `Throwable`.